tableauserverclient/models/server_info_item.py

```diff
@@ -32,7 +35,11 @@ def rest_api_version(self):
 
     @classmethod
     def from_response(cls, resp, ns):
-        parsed_response = fromstring(resp)
+        try:
+            parsed_response = fromstring(resp)
+        except xml.etree.ElementTree.ParseError as error:
+            warnings.warn("Unexpected response for ServerInfo: {}".format(resp))
```

Comment:
this isn't the general intended use for warnings -- they should be things thrown in tests or lints or ci/cd.

We should re-raise the exception, or if it isn't important just log and move on, but not a warnings.wan